Eatery Blue/Networking/Networking.swift

```diff
@@ -18,8 +18,11 @@ class Networking {
     let eateries: InMemoryCache<[Eatery]>
     let sessionId: InMemoryCache<String>
     let accounts: FetchAccounts
+    let baseUrl: URL
```

Comment:
Usually these should be sorted alphabetically, but if it isn't in the rest of the codebase then it's fine.